src/core/Akka.TestKit/TestKitBase_Receive.cs

```diff
@@ -456,7 +456,7 @@ public ValueTask<(bool success, MessageEnvelope envelope)> TryPeekOneAsync(TimeS
             [EnumeratorCancellation] CancellationToken cancellationToken = default)
         {
             var enumerable = ReceiveWhileAsync(filter, max, Timeout.InfiniteTimeSpan, msgs, cancellationToken)
-                .ConfigureAwait(false).WithCancellation(cancellationToken);
+                .WithCancellation(cancellationToken);
```

Comment:
We don't need `.WithCancellation()` here, token is already passed into the method.